docs/source/locale/es/LC_MESSAGES/install-kernel.po

```diff
@@ -3,106 +3,124 @@
 msgid ""
 "This information gives a high-level view of using Jupyter Notebook with "
 "different programming languages (kernels)."
 msgstr ""
+"Esta información le da un alto nivel de visualización a utilizar Jupyter "
+"Notebook con diferentes lenguajes de programación (kernels)."
 
 # 0b4e24aa51dd400782dfff162b605bb3
 #: ../../source/install-kernel.rst:13
 msgid "Are any languages pre-installed?"
-msgstr ""
+msgstr "Hay algún lenguaje preinstalado?"
 
 # ea86f10aa988483fbc44ed6fc2ed5c1c
 #: ../../source/install-kernel.rst:15
 msgid ""
 "Yes, installing the Jupyter Notebook will also install the `IPython "
-"<https://ipython.readthedocs.io/en/latest/>`_ :term:`kernel`. This allows"
-" working on notebooks using the Python programming language."
+"<https://ipython.readthedocs.io/en/latest/>`_ :term:`kernel`. This allows "
+"working on notebooks using the Python programming language."
 msgstr ""
+"Si, instalando Jupyter Notebook también instalará `IPython "
+"<https://ipython.readthedocs.io/en/latest/>`_ :term:`kernel`. Esto permite "
+"trabajar con notebooks usando el lenguaje de programación Python."
 
 # 94ea0ff4960049e8bffc34755b0c57d0
 #: ../../source/install-kernel.rst:20
 msgid "How do I install Python 2 and Python 3?"
-msgstr ""
+msgstr "Cómo instalo Python 2 y Python 3?"
 
 # 9e28ab95796d4facb934d8a06dbaf56c
 #: ../../source/install-kernel.rst:22
 msgid ""
-"To install an additional version of Python, i.e. to have both Python 2 "
-"and 3 available, see the IPython docs on `installing kernels "
+"To install an additional version of Python, i.e. to have both Python 2 and 3"
+" available, see the IPython docs on `installing kernels "
 "<https://ipython.readthedocs.io/en/latest/install/kernel_install.html>`_."
 msgstr ""
+"Para instalar una versión adicional de Python, es decir, tener Python 2 y "
+"Python 3 disponibles, vea la documentación de IPython en `installing kernels"
+" <https://ipython.readthedocs.io/en/latest/install/kernel_install.html>`_."
 
 # fc0d1726d8ff4504922a6164a4841b27
 #: ../../source/install-kernel.rst:27
 msgid "How do I install other languages like R or Julia?"
-msgstr ""
+msgstr "Cómo instalo otros lenguajes como R o Julia?"
 
 # 3eff7ec0dacb45dc9198c7dc21af8e4e
 #: ../../source/install-kernel.rst:29
 msgid ""
 "To run notebooks in languages other than Python, such as R or Julia, you "
-"will need to install additional kernels. For more information, see the "
-"full `list of available kernels`_."
+"will need to install additional kernels. For more information, see the full "
+"`list of available kernels`_."
 msgstr ""
+"Para ejecutar notebooks en lenguajes diferentes de Python, Cómo R o Julia, "
+"Debes instalar kernels adicionales. Para más información, vea la lista "
+"completa `lista de kernels disponibles`_."
 
 # a15c5c1d7def408f8bf85bb8b39608f6
 #: ../../source/install-kernel.rst:37
 msgid ":ref:`Jupyter Projects <subprojects>`"
-msgstr ""
+msgstr ":ref:`Proyectos de Jupyter <subprojects>`"
```

Comment:
I have a doubt here, when there's a ref where is the ref so it can be exactly the same? or isn't it actually connected to anything (I mean the reference)?

Reply:
I'm not quite sure. I did notice some issues with sphinx dealing with markup in the extracted translation strings in past message bundles. Maybe we can try it, see, and adapt?

Reply:
Merging so we can see what happens.